console-boot-logs/starlink-bootlog-1.txt

```diff
@@ -0,0 +1,159 @@
+INFO:    search_fip end,  slot: 1, result: 0
+INFO:    VALID code at slot 1 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 2
+INFO:    search_fip end,  slot: 2, result: 0
+INFO:    VALID code at slot 2 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 3
+INFO:    search_fip end,  slot: 3, result: 0
+INFO:    VALID code at slot 3 ref_ctr 0x5555 (21845)
+INFO:    search_fip start,  slot: 4
+INFO:    search_fip end,  slot: 4, result: 0
+INFO:    TERM code at slot 4
+INFO:    Chosen slot = 3
+INFO:    BL1: Get the image descriptor
+INFO:    BL1: Loading BL2
+INFO:    Loading image id=6 at address 0x30209000
+INFO:    Skip reserving region [base = 0x30209000, size = 0x90]
+INFO:    Image id=6 loaded at address 0x30209000, size = 0x90
+INFO:    cert_nv_ctr : 1
+INFO:    plat_nv_ctr : 0
+INFO:    Loading image id=1 at address 0x30209000
+INFO:    Image id=1 loaded at address 0x30209000, size = 0xf178
+NOTICE:  BL1: Booting BL2
+INFO:    Entry point address = 0x30209000
+NOTICE:  BL20: v1.3(debug):25afa823
+NOTICE:  BL20: Built : 16:55:25, Jul 17 2020
+NOTICE:  EMMC boot counter is 47
+NOTICE:  BL2: v1.3(debug):4ac84b3
+NOTICE:  BL2: Built : 15:39:21, Oct 15 2020
+NOTICE:  Evaluate 0x8110010 & 0x10 == 0x0 -> 0
+NOTICE:  Using default targetpack config
+Boot firmware v20190415-1 starting
+!*:#)$<@[|{!!("!%5!#!!"O-#A!!!"
+Boot firmware exiting normally
+NOTICE:  BL2: end TP
+NNOTICE:  BL31: v1.3(debug):4ac84b3
+NOTICE:  BL31: Built : 15:39:24, Oct 15 2020
+
+
+U-Boot 2016.05-00307-g16c388c (Jul 23 2020 - 22:19:05 +0000)
+
+DRAM:  1004 MiB
+MMC:   Fast boot:eMMC: 8xbit - div2
+stm-sdhci0: 0
+Using default environment
+
+In:    nulldev
+Out:   serial
+Err:   serial
+CPU ID: 0x00020100 0x09011015 0xb925838d
+Detected Board rev: #rev1_pre_production
+sdhci_transfer_data: Error detected in status(0x208000)!
+sdhci_transfer_data: Error detected in status(0x208000)!
+FIP1: 3 FIP2: 3
+BOOT SLOT B
+Net:   Net Initialization Skipped
+No ethernet found.
+
+                                                          *
+                                                 +         
+                                       +    +              
+                                +     +                    
+                           +      +                        
++ + + + +              +     +                             
+  +        +       +     +                                 
+     +       + +      +                                    
+        +   +      +                                       
+          +      + +                                       
+      +      +        +                                    
+   +       +    +        +                                 
+ +       +         +        +                              
++ + + + +             + + + + +                            
+
+Board: SPACEX CATSON UTERM
+
+Is production hardware? YES
+======================================
+= Type 'falcon' to stop boot process =
```

Comment:
Would you be able to try this?
It could allow offloading the Linux images and then getting the /etc/passwd and /etc/shadow file to try bruteforcing the password hashes.

Reply:
My best guess is there would be some kind of uBoot password required.